templates/workspaces/base/terraform/network/security.tf

```diff
@@ -53,25 +53,25 @@ resource "azurerm_network_security_rule" "deny_all_inbound_override" {
   source_port_range           = "*"
 }
 
-resource "azurerm_network_security_rule" "allow_inbound_within_services_subnet" {
+resource "azurerm_network_security_rule" "allow_inbound_within_workspace_vnet" {
```

Comment:
This opens up the whole "workspace" address space compared with just the "services" subnet of the workspace network. Has this been evaluated for the implications on the Airlock Review where I remember a discussion on which VM can access what?

Reply:
The VMs themselves for export have a MV NSG blocking access to other than the rules defined in the VM NSG. so this shouldn't affect this.